playground/AWS/AWS.AppHost/aspire-manifest.json

```diff
@@ -6,6 +6,9 @@
       "stack-name": "AspireSampleDevResources",
       "template-path": "app-resources.template",
       "references": [
+        {
```

Comment:
Looks like a bug

Reply:
Oh I didn't pick up that it was emitting twice!

Reply:
cc @normj

Reply:
Here is a fix https://github.com/dotnet/aspire/pull/5748